source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
+|         | This is where extensive I/O operations,                        |
+|         | large local, and all off-gaea transfers should be done.  These |
+|         | nodes are accessible via the :dfn:`es` cluster and the         |
+|         | :dfn:`dtn_f5_f6` partition.                                    |
++---------+----------------------------------------------------------------+
+
+.. _gaea-compute-nodes:
+
+Compute nodes
+=============
+
+Gaea consists of two clusters, C5 and C6.
+
+.. tab-set::
+
+  .. tab-item:: C5
+    :sync: C5
+
+    The C5 compute nodes consist of [2x] 64 core AMD EPYC Zen 2 CPUs, with two
+    hardware threads per physical core and 256 GiB of physical memory (2 GiB
+    per core). C5 supports up to the AVX-2 :abbr:`ISA (Insturction Set
+    Architechture)`.
+
+    .. figure:: /images/C5-ComputeNodeDiagram.png
+
+      Each C5 compute nodes has a total of 128 cores, in eight NUMA domains
+      per node.  Each group of four cores share an 16 MB L3 cache.  Each CPU
+      has eight lanes to the shared 256 GiB of node memory.
+
+  .. tab-item:: C6
+    :sync: C6
+
+    The C6 compute nodes consiste of [2x] 96 core AMD EPYC Zen 4 CPUs, with two
+    hardware threads per physical core and 384 GiB of physical memory (2 GiB
+    per core). C6 support up to the AVX-512 :abbr:`ISA (Insturction Set
+    Architechture)`.
+
+    .. figure:: /images/C6-ComputeNodeDiagram.png
+
+      Each C6 compute node has a total of 192 cores, in eight NUMA domains per
+      node.  Each group of six cores share a 48 MB L3 cache.  Each CPU has 12
+      lanes to the shared 384 GiB of physical memory (2 GiB per core).
+
+
+.. _gaea-login-nodes:
+
+Login nodes
+===========
+
+The Gaea login nodes have a similar architecture to the compute nodes.  Each
+compute cluster, C5 and C6, have a dedicated set of login nodes.
+
++----------------------+----------------------------+--------------------+
+| Host Names           | Node Configuration         | Associated Compute |
+|                      |                            | Cluster            |
++======================+============================+====================+
+| :regexp:`gaea5[1-8]` | 2x AMD EPYC 7662 64-core   | C5                 |
+|                      | (128 cores per node) with  |                    |
+|                      | 512 GiB of memory per node |                    |
++----------------------+----------------------------+--------------------+
+| :regexp:`gaea6[1-8]` | 2x AMD EPYC 9654 96-core   | C6                 |
+|                      | (192 cores per node) with  |                    |
+|                      | 512 GiB of memory per node |                    |
++----------------------+----------------------------+--------------------+
+
+.. _gaea-dtn-nodes:
+
+Data transfer nodes
+===================
+
+The data transfer nodes (DTN) is where extensive I/O operations, large local,
+and all off-gaea transfers should be done.  The :abbr:`DTN (Data Transfer
```

Comment:
Recommend:

All extensive I/O operations, large local,
and all off-gaea transfers should be done in data transfer nodes (DTN).

Reply:
Updated in a new push.